### PR TITLE
added rhoda

### PR DESCRIPTION
**Pull request by rrbanda**
##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New config Pull Request
- New role Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by tonykay — COMMENTED** [review]
Looks good to me, tagging @wkulhanek and @newgoliath in deference to their familiarity with ocp_workloads

File: ansible/ocp4_workload_cnd_rhoda_operator/tasks/workload.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  loop: "{{ range(1, ocp4_workload_authentication_htpasswd_user_count | int + 1, 1) | list }}"
+
+- name: give user admin privileges in namespace
+  k8s:
+    state: present
+    namespace: "user{{ item }}{{ ocp4_workload_cnd_rhoda_operator_namespace }}"
+    definition: "{{ lookup('template', role_path ~ '/templates/namespace-admin-rb.j2' ) | from_yaml }}"
+  loop: "{{ range(1, ocp4_workload_authentication_htpasswd_user_count | int + 1, 1) | list }}"
+
+- name: Install RHODA Operator
```
Comment (newgoliath) [inline ansible/ocp4_workload_cnd_rhoda_operator/tasks/workload.yml:60]: fix name of task to match dbaas operator

File: ansible/ocp4_workload_cnd_rhoda_operator/tasks/workload.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  vars:
+    install_operator_action: install
+    install_operator_name: rhoda-operator
+    install_operator_namespace: "{{ ocp4_workload_cnd_rhoda_namespace }}"
+    install_operator_channel: "{{ ocp4_workload_cnd_rhoda_operator_subscription_channel }}"
+    install_operator_catalog: "{{ ocp4_workload_cnd_rhoda_catalogsource_name }}"
+    install_operator_packagemanifest_name: "{{ ocp4_workload_cnd_rhoda_install_operator_name }}"
+    install_operator_automatic_install_plan_approval: true
+    install_operator_csv_nameprefix: rhoda-operator
+    install_operator_starting_csv: ""
```
Comment (newgoliath) [inline ansible/ocp4_workload_cnd_rhoda_operator/tasks/workload.yml:34]: i think you should be opinionated here.  no external dependencies.

File: ansible/ocp4_workload_cnd_rhoda_operator/templates/subscription.yaml.j2 [hunk 3]
```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1alpha1
```
Comment (newgoliath) [inline ansible/ocp4_workload_cnd_rhoda_operator/templates/subscription.yaml.j2:1]: Omit this file if you're not using it.

**Review by newgoliath — APPROVED** [review]

**Comment by veniceofcode** [comment]
hi @wkulhanek Can you please review and approve ?

**Review by newgoliath — APPROVED** [review]

**Pull request closed without merging**